changelogs/v5.json

```diff
@@ -1,6 +1,152 @@
 {
   "$schema": "./schema.json",
   "releases": [
+    {
+      "version": "5.0.0-rc.0",
+      "date": "2021-01-07",
+      "description": "This is a release candidate for our next major version which contains new features and bug fixes. We've also added a package that provides Angular modules for Core package's web componnets.",
+      "feat": [
+        {
+          "issue": 5337,
+          "title": "Export icon names from the entrypoint of @clr/core/icon",
+          "project": "Core",
+          "scope": "Icons"
+        },
+        {
+          "issue": 5288,
+          "title": "Angular modules for Core components",
+          "project": "Angular",
+          "scope": "Build"
+        },
+        {
+          "issue": 4839,
+          "title": "Volume 18 icons",
+          "project": "Core",
+          "scope": "Icons"
+        },
+        {
+          "issue": 4502,
+          "title": "Core i18n",
+          "project": "Core",
+          "scope": "i18n"
+        },
+        {
+          "issue": 5389,
+          "title": "Allow env flag set for core logging",
+          "project": "Core",
+          "scope": "Build"
+        },
+        {
+          "issue": 5200,
+          "title": "Overlay component",
+          "project": "Core",
+          "scope": "Dropdown"
+        },
+        {
+          "issue": 5267,
+          "title": "Add eslint rule for detecting and fixing clr icons",
+          "project": "Clarity Adoption",
+          "scope": "Build"
+        }
+      ],
+      "fix": [
+        {
+          "issue": 5428,
+          "title": "Positioning of overlay backdrop in demos in safari",
+          "project": "Core",
+          "scope": "Modal"
+        },
+        {
+          "issue": 5424,
+          "title": "Improve alert examples aria label",
+          "project": "Core",
+          "scope": "Alerts"
+        },
+        {
+          "issue": 5423,
+          "title": "Make the eslint rule for icons work with multiple root elements",
+          "project": "Clarity Adoption",
+          "scope": "Build"
+        },
+        {
+          "issue": 5409,
+          "title": "Prevent Core and UI text collisions",
+          "project": "Core",
+          "scope": "UI"
+        },
+        {
+          "issue": 5405,
+          "title": "Fix inherit styles",
+          "project": "Core",
+          "scope": "Icons"
+        },
+        {
+          "issue": 5355,
+          "title": "Fix not found icon loading state",
+          "project": "Core",
+          "scope": "Icons"
+        },
+        {
+          "issue": 5400,
+          "title": "Core themes",
```

Comment:
Core themes is a feature, not a fix. It also had several breaking changes to tokens and the custom properties available to the core components.